internal/controller/validator.go

```diff
@@ -139,8 +141,45 @@ func (r *DataProtectionApplicationReconciler) ValidateDataProtectionCR(log logr.
 				appliedBackupSyncPeriod, appliedGarbageCollectionPeriod,
 			)
 		}
-		// TODO should also validate that BSL backupSyncPeriod is not greater or equal to nonAdmin.backupSyncPeriod
-		// but BSL can not exist yet when we validate the value
+
+		defaultBSLIndex := -1
+		bslsInOADPNamespace := &velerov1.BackupStorageLocationList{}
+		r.List(r.Context, bslsInOADPNamespace, client.InNamespace(r.NamespacedName.Namespace))
+		for index, bsl := range bslsInOADPNamespace.Items {
+			if bsl.Spec.Default {
+				defaultBSLIndex = index
+				break
+			}
+		}
+		if defaultBSLIndex >= 0 {
+			defaultBSL := bslsInOADPNamespace.Items[defaultBSLIndex]
+			defaultBSLSyncPeriodErrorMessage := "default BSL spec.backupSyncPeriod (%v) can not be greater or equal spec.nonAdmin.backupSyncPeriod (%v)"
+			if defaultBSL.Spec.BackupSyncPeriod != nil {
+				if appliedBackupSyncPeriod <= defaultBSL.Spec.BackupSyncPeriod.Duration {
+					return false, fmt.Errorf(
+						defaultBSLSyncPeriodErrorMessage,
+						defaultBSL.Spec.BackupSyncPeriod.Duration, appliedBackupSyncPeriod,
+					)
+				}
+			} else {
+				if r.dpa.Spec.Configuration.Velero.Args != nil && r.dpa.Spec.Configuration.Velero.Args.BackupSyncPeriod != nil {
```

Comment:
changing this value in DPA does not recreate NAC Pod
 
is this a problem?

Reply:
That means that NAC won't be pulling in the latest updates, right ?

Reply:
uhum

Reply:
alright, I think I fixed, can you test

IBM is currently down, could not test in my cluster